pkg/client/client_test.go

```diff
@@ -482,6 +482,60 @@ func TestClient_UpdateDocument_ServerUnreachable(t *testing.T) {
 	require.True(t, testPassed)
 }
 
+func TestClient_DeleteDocument(t *testing.T) {
+	srvAddr := randomURL()
+
+	srv := startEDVServer(t, srvAddr)
+
+	waitForServerToStart(t, srvAddr)
+
+	client := New("http://" + srvAddr + "/encrypted-data-vaults")
+
+	validConfig := getTestValidDataVaultConfiguration()
+	vaultLocationURL, err := client.CreateDataVault(&validConfig)
+	require.NoError(t, err)
+
+	vaultID := getVaultIDFromURL(vaultLocationURL)
+
+	_, err = client.CreateDocument(vaultID, getTestValidEncryptedDocument(testJWE))
+	require.NoError(t, err)
+
+	err = client.DeleteDocument(vaultID, testDocumentID)
```

Comment:
Can you add a call to resolve deleted document here? I saw that you have this test at store level (great!) but it is also nice to have it at client level.